csrankings-8.csv

```diff
@@ -756,6 +757,7 @@ Stefan Wolfgang Pickl,Bundeswehr University Munich,https://www.unibw.de/comtessa
 Stefan Woltran,TU Wien,https://www.dbai.tuwien.ac.at/staff/woltran,NOSCHOLARPAGE
 Stefan Wrobel,University of Bonn,https://mlai.cs.uni-bonn.de/people/stefan-wrobel,NOSCHOLARPAGE
 Stefan Wuchty,University of Miami,http://www.as.miami.edu/csc/people/faculty/dr-stefan-wuchty,NOSCHOLARPAGE
+Stefania Serafin,Aalborg University,https://vbn.aau.dk/en/persons/107881,OPdehucAAAAJ

```

Comment:
Not listed here: https://vbn.aau.dk/da/organisations/institut-for-datalogi/persons/?page=4

Reply:
Bump @henningpohl

Reply:
Sorry, missed that one. I removed Stefania.